src/components/Policy/__tests__/PolicyWizard.test.tsx

```diff
@@ -23,6 +23,7 @@ describe('src/components/Policy/PolicyWizard', () => {
         jest.useFakeTimers();
         render(
             <PolicyWizard
+                data-ouia-component-id={ 'pendo-policy-wizard' }
```

Comment:
Why is this needed?

Reply:
Pendo is missing selectors for the wizard -- see attached ticket https://issues.redhat.com/browse/RHINENG-7853